From ea4d0a4dc74b54053ae12eadcef9bf474b29b838 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Mon, 31 Jul 2023 15:04:36 +0200
Subject: [PATCH 0492/1141] arm64: dts: st: add features-domains for i2s on
 stm32mp251

Add features-domains in I2S nodes for STM32MP251.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I873409e24d2f125065c8f4af4cf4280367ba5b5d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/324537
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 7534f02e63da..5da4ce6c1975 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -703,6 +703,7 @@ i2s2: audio-controller@400b0000 {
 				dmas = <&hpdma 51 0x43 0x12 0>,
 					<&hpdma 52 0x43 0x21 0>;
 				dma-names = "rx", "tx";
+				feature-domains = <&rifsc STM32MP25_RIFSC_SPI2_ID>;
 				power-domains = <&RET_PD>;
 				status = "disabled";
 			};
@@ -734,6 +735,7 @@ i2s3: audio-controller@400c0000 {
 				dmas = <&hpdma 53 0x43 0x12 0>,
 					<&hpdma 54 0x43 0x21 0>;
 				dma-names = "rx", "tx";
+				feature-domains = <&rifsc STM32MP25_RIFSC_SPI3_ID>;
 				power-domains = <&RET_PD>;
 				status = "disabled";
 			};
@@ -1122,6 +1124,7 @@ i2s1: audio-controller@40230000 {
 				dmas = <&hpdma 49 0x43 0x12 0>,
 					<&hpdma 50 0x43 0x21 0>;
 				dma-names = "rx", "tx";
+				feature-domains = <&rifsc STM32MP25_RIFSC_SPI1_ID>;
 				power-domains = <&RET_PD>;
 				status = "disabled";
 			};
-- 
2.39.2

